test/parallel/test-crypto-dh.js

```diff
@@ -285,4 +285,4 @@ ecdh5.setPrivateKey(cafebabeKey, 'hex');
 // invalid test: curve argument is undefined
 assert.throws(() => {
   crypto.createECDH();
-}, /^TypeError: "curve" argument should be a string$/);
+}, /^TypeError: ECDH curve name must be a string$/);
```

Comment:
Why is this semver-major? It looked like an internal organization that made no API changes, except for this error message change. Can the error message be made backwards compatible, so we can backport everything else, and just do a semver-major PR afterwards to update the error message for 8.x?